src/octoprint/static/js/app/viewmodels/settings.js

```diff
@@ -801,7 +801,6 @@ $(function () {
                     }
                 })
                 .css({
-                    "width": "auto",
```

Comment:
No clue what this was for, and it causes issues with the sizing of the dialog. Git shows it going way back to when the dialog was implemented, I noticed no adverse effects from removing it.

Reply:
Let's just hope for the best then, I have no clue anymore either.